README.md

```diff
@@ -6,8 +6,8 @@ A typical TODO item should have:
  - title
```

Comment:
I would add _non empty_ to title. Also, what do you think about distinguishing TODO items with the same `title`? This problem can be solved by introducing unique ids to every TODO. But personally I think that it's better to store timestamp inside TODO item — time of creation.

Reply:
I agree with you about storing timestamps to distinguish TODOs with the same titles.
Should it be mentioned somehow in this README?
It seems to me that timestamps are not something user should be aware of.

Reply:
I think this can be mentioned. Of course, we can make it internal detail of implementation. But this can also be part of UI, if we add command like `show 10 latest created TODO items`. In issue #3 you don't need to care about timestamp, it requires real world and we don't care about real world for now. At some point in future I will create issue about adding timestamps.